app/controllers/concerns/hydra/collections/selects_collections.rb

```diff
@@ -11,16 +11,35 @@ def access_levels
     { read: [:read, :edit], edit: [:edit] }
   end
 
-  # add one of the following methods as a before filter on any page that shows the form_for_select_collection
+  ##
+  # Return list of collections for which the current user has read access.
+  # Add this or find_collections_with_edit_access as a before filter on any page that shows the form_for_select_collection
+  #
+  # @return [Array<SolrDocument>] Solr documents for each collection the user has read access
   def find_collections_with_read_access
     find_collections(:read)
   end
 
-  def find_collections_with_edit_access
+  ##
+  # Return list of collections for which the current user has edit access.  Optionally prepend with default
+  # that can be used in a select menu to instruct user to select a collection.
+  # Add this or find_collections_with_read_access as a before filter on any page that shows the form_for_select_collection
+  #
+  # @param [TrueClass|FalseClass] :include_default if true, prepends the default_option; otherwise, if false, returns only collections
+  # @param [Fixnum] :default_id for select menus, this is the id of the first selection representing the instructions
+  # @param [String] :default_title for select menus, this is the text displayed as the first item serving as instructions
+  #
+  # @return [Array<SolrDocument>] Solr documents for each collection the user has edit access, plus optional instructions
+  def find_collections_with_edit_access(include_default=false, default_id=-1, default_title="Select collection...")
     find_collections(:edit)
+    default_option = SolrDocument.new(id: default_id, title_tesim: default_title)
```

Comment:
I thought about that originally.  I went with creating the SolrDocument in find_collection_with_edit_access because it returns an Array of SolrDocuments.  How would the caller know this (except that now it is documented ;)  I was providing a way to extend the Array without having to know what was inside.  The select_tag needs to be able to call an id and title method.  If you don't create a SorDocument as the default_option, you might run into troubles.  Can you give an example of a case where the caller would want to create something other than a SolrDocument?

That said, it would function the way you suggest, so I am happy to change it if that is what you prefer.  If you want find_collections(:edit) to be last, I can make that change without changing the parameters.